### PR TITLE
adds encode sans and minor link coloring change to be same as static …

### PR DESCRIPTION
**Pull request by RobertWSaunders**
*Fixes ticket #42 *

### Changes proposed in this PR:
- adds encode sans
- updates link colors

### CODE REVIEW

File: client/assets/semantic-ui/site/globals/site.variables [hunk 1]
```diff
@@ -9,11 +9,11 @@
...
 @fontSmoothing     : antialiased;
 
-@headerFont        : @fontName, 'Helvetica Neue', Arial, Helvetica, sans-serif;
-@pageFont          : @fontName, 'Helvetica Neue', Arial, Helvetica, sans-serif;
+@headerFont        : @fontName, Arial, 'Helvetica Neue', Helvetica, sans-serif;
+@pageFont          : @fontName, Arial, 'Helvetica Neue', Helvetica, sans-serif;
 
-@googleFontName    : @fontName;
-@importGoogleFonts : false;
+@googleFontName    : 'Encode+Sans';
```
Comment (zackharley) [inline client/assets/semantic-ui/site/globals/site.variables]: Shouldn't we still use `@fontName` here?

File: client/components/utils/LoginForm.js [hunk 2]
```diff
@@ -46,7 +46,7 @@ class LoginForm extends Component {
 					<Link to="/reset-password">Forgot password?</Link>
 					<Divider/>
 					<p>
-						Havent applied yet? <Link to="/apply">Apply Here</Link>
+						Haven't applied yet? <Link to="/apply">Apply Here</Link>
```
Comment (zackharley) [inline client/components/utils/LoginForm.js]: Good catch! 👍

File: package.json [hunk 3]
```diff
@@ -63,6 +63,7 @@
     "favicons-webpack-plugin": "0.0.7",
     "file-loader": "^1.1.5",
     "html-webpack-plugin": "^2.30.1",
+    "jsx-ast-utils": "^2.0.1",
```
Comment (zackharley) [inline package.json]: What's this?

File: client/assets/semantic-ui/site/globals/site.variables (hunk 1 above)
Comment (RobertWSaunders) [reply client/assets/semantic-ui/site/globals/site.variables]: CSS requires `Encode Sans` whereas the google fonts api requires `Encode+Sans` so they have to be different unfortunately.

Comment (zackharley) [reply client/assets/semantic-ui/site/globals/site.variables]: Okay, thanks!

**Pull request merged**